grafana/grafana
# Tracing: Add trace to metrics config behind feature toggle

**What this PR does / why we need it**:
Adds the alpha config for trace to metrics behind a feature flag with a hard-coded metrics query. 

It also refactors how links are displayed in the trace viewer to show all links (to traces, to metrics, and to logs) from a menu. 

This is behind a feature flag and won't be enabled for any customers. Future PRs will:
1. Make the query configurable
2. Add support for writing multiple metrics queries
3. Add support for interpolating span attributes into the query

**Which issue(s) this PR fixes**:



Fixes #

**Special notes for your reviewer**:

Testing instructions
- Enable the `traceToMetrics` feature flag
- Turn on trace to metrics in the datasource config for a tracing datasource
- Check that the link appears and opens the linked metrics datasource


![Screen Shot 2022-04-27 at 10 46 55 AM](https://user-images.githubusercontent.com/12838032/165579947-36f7157c-c08b-4748-8d45-9e7c2778435e.png)
